src/index.js

```diff
@@ -0,0 +1,169 @@
+const logTempText = () => {
+  displayTempId.textContent = `${temperatureNumber}°F`;
+}
+const changeTextColor = () => {
+  if (temperatureNumber < 50) {
+    displayTempId.style.color = "aqua";
+  } else if (temperatureNumber >= 50 && temperatureNumber <= 59) {
+    displayTempId.style.color = "limegreen";
+  } else if (temperatureNumber >= 60 && temperatureNumber <= 69) {
+    displayTempId.style.color = "gold";
+  } else if (temperatureNumber >= 70 && temperatureNumber <= 79) {
+    displayTempId.style.color = "orange";
+  } else {
+    displayTempId.style.color = "red";
+  }
+};
+
+const displayLandscape = document.getElementById('landscape');
+const changeLandscape = (temp) => {
+  if (temp > 79) {
+    return '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp >= 70 && temp <= 79) {
+    return '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp >= 60 && temp <= 69) {
+    return '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (temp <= 59) {
+    return '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const logLandscape = (temp) => {
+  displayLandscape.textContent = `${changeLandscape(temp)}`
+};
+
+const weatherIcon = (imgURL) => {
+  let img=document.createElement('img');
+  img.src = imgURL;
+}
+
+
+const displaySky = document.getElementById('sky');
+const dropdownValue = document.getElementById('weather-dropdown');
+const changeSky = (skyDrop) => {
+  if (skyDrop === 'sun') {
+    return '☁️ _ ☁️ _ _ _ ☀️ _ _ _ ☁️ _ ☁️';
+  } else if (skyDrop === 'clouds') {
+    return '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skyDrop === 'rain') {
+    return '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skyDrop === 'snow') {
+    return '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  } else {
+    return ' ';
+  }
+};
+
+
+
+
+dropdownValue.addEventListener('change', function handleChange () {
+  let newSky = changeSky(dropdownValue.options[dropdownValue.selectedIndex].value);
+  displaySky.textContent = `${newSky}`;
+  display
+});
+
+logTempText();
+changeTextColor();
+logLandscape(temperatureNumber);
+
+const currentWeather = document.getElementById('current-temp-button');
+currentWeather.addEventListener("click", function () {
+  if (textBox.value !== '') {
+    findLocation(textBox.value);
+  } else {
+    findLocation('San Diego, CA, USA');
+  }
+});
+
+const upButton = document.getElementById('increase-temp-button');
+upButton.textContent = '🔥';
+upButton.addEventListener("click", function() {
+  temperatureNumber += 1;
+  logTempText();
+  changeTextColor();
+  logLandscape(temperatureNumber);
+});
+
+const downButton = document.getElementById('decrease-temp-button');
+downButton.innerText = '❄️';
+downButton.addEventListener("click", function() {
+  temperatureNumber -= 1;
+  logTempText();
+  changeTextColor();
+  logLandscape(temperatureNumber);
+});
+
+const resetCity = document.getElementById('reset-city-button');
+resetCity.addEventListener("click", function() {
+  currentCity = 'San Diego, CA, USA';
+  document.getElementById('display-city').textContent = currentCity;
+  document.getElementById('text-box').value='';
+})
```

Comment:
There's an issue with wave 04 (calling the apis), the requirements specify that your code should get the weather of the currently displayed city name & when the  <button> element is clicked updates and displays the realtime temperature of the currently displayed city name. Currently your code is not resetting the temperature of the displayed city.